### PR TITLE
Version update

### PR DESCRIPTION
**Pull request by LayaTaal**
I had some trouble bringing up the local environment on my computer this morning so I thought I'd run through some updates. I've tested this on my environment, but I'd like for others to run through the rebuild sequence and and make sure everything still works as expected.

If there are any other core updates you think we should make here, let's address those as well.

## Building this Branch

`cd` to the project directory and then follow the instructions below.

1. `rm -rf node_modules` from root directory
2. `nvm use 12.18.3`. If it's not installed, `nvm install 12.18.3`. Assuming you have Node Version Manager..if you don't, [get it!](https://github.com/nvm-sh/nvm/blob/master/README.md)
3. Reinstall node modules `npm install`
5. `npm run dev` to bring up your local server on `localhost:3333`

### Adds
- Node Version Manager to establish as standard version

### Modifies
- Node version to 12.18.3
- NextJs to latest
- Dependencies of both packages
- Cosmetic update to .gitignore.

### Removes
- Removes two packages which are no longer used, `Victory` and `SendMail`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,19 +5,17 @@
     "export": "next export",
     "deploy": "npm run build && npm run export",
```
Comment (nholden) [inline package.json:6]: Do we use the `deploy script`? If not, we should dump it. If we do use it, we'll need to update it to use yarn if we move forward with that change.

**Comment by LayaTaal** [comment]
Hey @nholden thanks for the feedback. I think you are right about the switch to yarn, I don't see any particular benefit to it either and no reason to have to update our builds.

I think there will still be fair number of dependency updates (npm flagged something like 340 audits), along with those downstream. I feel like our best bet here is to let this PR go up on a staging branch for a while so we can all play with it and confirm there are no issues.

I didn't realize that about `next-optimized-images`, that's really strange that it doesn't work with the way NextJS prefers to do things. I haven't played around with it in this branch to confirm, but are we sure it still works this way? I also noticed they are releasing v3 soon, so we could always hold off on any changes related to this plugin until that is released and we can test it.

What do you think?

I can go ahead and revert the update to yarn and the image relocation, if you agree, and I will remove the deploy script as well. Nice catch there!

Thanks for the review!

**Comment by nholden** [comment]
> I think there will still be fair number of dependency updates (npm flagged something like 340 audits), along with those downstream. I feel like our best bet here is to let this PR go up on a staging branch for a while so we can all play with it and confirm there are no issues.

Makes sense to me! 👍 

> I didn't realize that about next-optimized-images, that's really strange that it doesn't work with the way NextJS prefers to do things. I haven't played around with it in this branch to confirm, but are we sure it still works this way?

Yes, we're still using `next-optimized-images` [as configured here](https://github.com/texas-justice-initiative/website-nextjs/blob/b455ba2f7407b614657f6d66c2cb80207eb2d73b/next.config.js#L41-L51), and we need to `require` those images to get the benefits of `next-optimized-images` as described in [the docs](https://github.com/cyrilwanner/next-optimized-images#usage).

[Next.js's static file serving](https://nextjs.org/docs/basic-features/static-file-serving) and `next-optimize-images` are fundamentally different approaches, so it doesn't surprise me too much that they look different. The former serves up the image exactly as it exists in the repository, and the latter performs all sorts of optimizations on the image. I'd be open to exploring getting rid of `next-optimize-images` and taking the simpler approach, but I think it's a big enough change that we'd want to handle that on its own.

**Comment by LayaTaal** [comment]
> [Next.js's static file serving](https://nextjs.org/docs/basic-features/static-file-serving) and `next-optimize-images` are fundamentally different approaches, so it doesn't surprise me too much that they look different. The former serves up the image exactly as it exists in the repository, and the latter performs all sorts of optimizations on the image. I'd be open to exploring getting rid of `next-optimize-images` and taking the simpler approach, but I think it's a big enough change that we'd want to handle that on its own.

Agreed. I will pull out those changes and ping you for another look once it's ready.

File: package.json [hunk 2]
```diff
@@ -3,21 +3,18 @@
     "dev": "next -p 3333",
     "build": "next build",
     "export": "next export",
-    "deploy": "npm run build && npm run export",
```
Comment (nholden) [inline package.json:6]: Looks like we still need this script to deploy to Netlify.

**Comment by nholden** [comment]
I clicked through a handful of the pages and tested some of the interactions on the explore page and Covid map, and everything looked great! LMK if you have anything in particular you'd like me to test out.

**Comment by nholden** [comment]
I definitely think it's worth upgrading our dependencies. Given that this PR has grown a little stale, I think we should consider starting fresh. I also think we should consider opening smaller PRs that upgrade as few dependencies as possible at once to reduce risk.

**Pull request closed without merging**